docs/user/trusted-publishers/adding-a-publisher.md

```diff
@@ -106,6 +106,11 @@ each.
 
 === "GitLab CI/CD"
 
+    !!! note
+
+        Currently, only projects hosted on https://gitlab.com are supported. Self-managed
```

Comment:
Nitpick for linkification (similar for other additions below):

```suggestion
        Currently, only projects hosted on <https://gitlab.com> are supported. Self-managed
```

Reply:
fixed